hotkey.js

```diff
@@ -14,10 +14,64 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function isElementIsIgnoredInRange(parent, node) {
+  if (node.tagName === 'BODY') return null
+  if (parent === node) return null
+  if (node.hasAttribute('data-keyux-ignore-hotkeys')) return node
+
+  return isElementIsIgnoredInRange(parent, node.parentNode)
+}
+
+function findNonIgnoredElement(activeElement, elements) {
+  for (let element of elements) {
+    if (isElementIsIgnoredInRange(activeElement, element)) continue
+    return element
+  }
+
+  return null
+}
+
 function checkHotkey(where, code, overrides) {
   let codeOverride = overrides[code]
   if (Object.values(overrides).includes(code) && !codeOverride) return false
-  return where.querySelector(`[aria-keyshortcuts="${codeOverride || code}" i]`)
+
+  if (
+    where.activeElement.getAttribute('aria-keyshortcuts') ===
+    (codeOverride || code)
+  ) {
+    return where.activeElement
+  }
+
+  if (where.activeElement.getAttribute('data-keyux-hotkeys')) {
+    let el = where.querySelector(
+      `#${where.activeElement.getAttribute('data-keyux-hotkeys')}`
+    )
+
+    if (el) {
```

Comment:
We don't need this check. Let's save size by requiring pointing to the wrapper

Reply:
done